test/images/hostexec/Dockerfile

```diff
@@ -21,5 +21,5 @@ CROSS_BUILD_COPY qemu-QEMUARCH-static /usr/bin/
 # - iproute2: includes ss used in NodePort tests
 RUN apk --update add curl netcat-openbsd iproute2 && rm -rf /var/cache/apk/*
 
-# wait forever
-CMD rm -f /fifo && mkfifo /fifo && exec cat </fifo
```

Comment:
`while true; do sleep 1d; done`?

Reply:
Done. Will need to push a new image and update the tests.